OpenRA.Game/Actor.cs

```diff
@@ -129,24 +136,34 @@ internal Actor(World world, string name, TypeDictionary initDict)
 				.Select(pair => new SyncHash(pair.First, pair.Second(pair.First)));
 		}
 
-		Rectangle DetermineBounds()
+		Rectangle DetermineRenderBounds()
+		{
+			var size = TraitsImplementing<IAutoSelectionSize>().Select(x => x.SelectionSize(this)).FirstOrDefault();
```

Comment:
Using `IAutoSelectionSize` for the render bounds feels really wrong.  Can we introduce a new interface, even if it just returns the same values for now?